### ISSUE TITLE
[Bug] Image tag doesn't work in execp

### ISSUE THREAD

**Issue by Microcrap** [brndnmtthws/conky#903]
**Step to reproduce :**

1. Creat a conky.
2. In it, execp a bash script.
3. In you bash script, try to push an image example (this example display distro version and replace debian or ubuntu texte by an appropriate image (ex ubuntu_logo.png) localised in ~/.conky/conky-perso/SysInfos/:

`gcc --version | grep -Eio "ubuntu|debian|arch|red hat" | head -1  | tr -d '\n' | tr '[:upper:]' '[:lower:]' | sed -E "s#ubuntu|debian#\${image ~/.conky/conky-perso/SysInfos/&_logo.png -s 25,25}#i"`

**Comment by lasers** [comment]
The code is slightly hard to parse. Will it work if you use `lsb_release -is` instead?

**Comment by lasers** [comment]
Alternative: See `/etc/os-release`.

**Comment by lasers** [comment]
You need to use `-s 25x25`, not `-s 25,25`. This works okay here! :-)

**Comment by Microcrap** [comment]
> Alternative: See `/etc/os-release`.

I don't know wich is better for multi-distro compatibility. I use it on remote servers (from what i have googled, gcc seems to have a +- similar response from one distro to another).

> You need to use `-s 25x25`, not `-s 25,25`. This works okay here! :-)

Ah nice the image is displayed ! Thanks.
But is it possible to has the image in the text same as an emoji (without a fixed position on the background) ?
I want display the distro logo between the country and the server name on the following conky :
![Screenshot_2019-11-06_00-44-37](https://user-images.githubusercontent.com/56029538/68255947-f78e1180-002e-11ea-94af-6667e8e67438.png)

**Comment by lasers** [comment]
I'm not versed with conky nor do I know C/C++. I am involved only with issues or pull requests.

It looks like it's not possible according to...
>Conky does  not  make  any attempt to adjust the position (or any other formatting) of images, they are just rendered as per the arguments passed. The only reason $image is part of the conky.text section, is to allow for runtime modifications, through $execp $lua_parse, or some other method.

Sorry.

My suggestion would be to try [imagemagick](http://www.imagemagick.org/Usage/text/) to create+combine texts+images (country, distro logo, server name, etc). Make it come out the way you want it... and have conky displaying that image.
